NewSemVer.ps1

```diff
@@ -78,7 +81,7 @@ Update-AssemblyVersionAttributes $version
 # Rebuild the solution to bake the new version into the assemblies. This is using the default
 # VS2017 location for MSBuild, so change it as appropriate.
 # C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin
-$msBuildExe = (${env:ProgramFiles(x86)}, 'Microsoft Visual Studio', '2017', 'Enterprise', 'MSBuild', '15.0', 'bin', 'msbuild.exe') `
+$msBuildExe = (${env:ProgramFiles(x86)}, 'Microsoft Visual Studio', 'Preview', 'Enterprise', 'MSBuild', '15.0', 'bin', 'msbuild.exe') `
```

Comment:
So is there a preview version necessary to make something work? If so, can you add a comment about what that is exactly and a note to move to the release version when it's available? I thought this was all supposed to be supported with the latest release.

Reply:
Well preview is what I have (Version 15.3.0 Preview 7.0) but really a release version of 15.3 or any version of 15.4 *should* work. It's just a bummer since the string changes, plus I currently have both community and Enterprise installed I think, so that also changes the string.

Reply:
Definitely older versions will *not* work, so a comment is worthwhile anyway, because if you try to build this project with, say, 15.1, it'll complain that it has no idea how to do that. So yeah, I might leave the string all messy since that has to be fixed for each dev environment anyway depending on what you have installed, but I'll specify in comments that 15.3 or newer should be used.

Reply:
Yeah I guess it's kind of unfortunate to have to come up with the path to msbuild in this script. I don't have Enterprise installed, for example. I'm ok with having to point it at the path to msbuild now that I know I need to.

Reply:
I looked at example code for trying to automatically figure out msbuild location, but it seemed super complicated since there are tons of different locations across versions and SKUs, so I just went with the "...so change it as appropriate" lazy comment. I only expect a few of us who actually ever push NuGet packages to have to use this script anyway. The other option was to just assume the caller has made sure the correct `msbuild.exe` is in the PATH, which I had also considered and could be an option going forward.